vignettes/Deployment.Rmd

```diff
@@ -0,0 +1,79 @@
+---
+title: "Deployment"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Deployment}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include=FALSE}
+# Load packages.
+library(fontawesome)
+
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+
+To reap all the advantages of the `riskassessment` application, an organization needs to frequently execute and store results derived from within the app, including but not limited to:
+
+1. uploading new package info
+2. publishing reviewer comments
+3. storing metric weights
+4. making package usage decisions for GxP environments
+5. etc
+
+To accomplish this, the app automatically creates and populates a `SQLite` database, stored locally in the application's root directory when launched for the first time. In addition, a `loggit.json` file records some rudimentary logs of actions performed in the app and is also stored in the root directory. As you might expect, some deployment environments come with limitations that are not amicable to said storage method! Spoiler alert - shinyapps.io is one of them. This article will provide a run down of the following deployment options and supply helpful tips as it pertains to `riskassessment`:
```

Comment:
Similarly, instructions on how to configure the location of `loggit.json` though I am not sure this is currently possible?

Reply:
Accurate assumption 👍🏼

Reply:
Opened a new issue to add that. For now, marking this convo as "resolved".